Networking/NetworkingTests/Remote/OrdersRemoteTests.swift

```diff
@@ -462,6 +462,21 @@ final class OrdersRemoteTests: XCTestCase {
         assertEqual(received, expected)
     }
 
+    func test_update_order_when_payment_method_id_and_title_passed_then_request_parameters_set() throws {
```

Comment:
Thanks for covering the `OrdersRemoteTests` accounting for the new properties! 💯